SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/ReferenceCountedLockingPolicyTest.cpp

```diff
@@ -18,22 +31,42 @@ struct TestData
 {
 	UWorld* TestWorld;
 	TMap<FName, AActor*> TestActors;
+	TMap<AActor*, TArray<TPair<ActorLockToken, FString>>> TestActorToLockingTokenAndDebugStrings;
```

Comment:
🤢

Reply:
Does this emoji mean you want this changed? It's a verbose type and name I agree, but more manageable and easier to debug than having multiple maps imo

Reply:
Generally I'd prefer breaking down nested container type with `using` but as this usage is well contained, this is fine.